<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>HttpProxyLoginWindow</class>
 <widget class="QDialog" name="HttpProxyLoginWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone - Local HTTP Proxy</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Your local HTTP proxy uses authentication.
Please enter your login and password in order to use the local HTTP proxy.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="addressLineEdit" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="addressLabel" >
     <property name="text" >
      <string>HTTP Proxy Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="portLineEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="portLabel" >
     <property name="text" >
      <string>HTTP Proxy Port Number:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="loginLineEdit" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="loginLabel" >
     <property name="text" >
      <string>HTTP Proxy Login:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="passwordLineEdit" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="passwordLabel" >
     <property name="text" >
      <string>HTTP Proxy Password:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>213</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="wengophone.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>HttpProxyLoginWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>252</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>HttpProxyLoginWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>334</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>304</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
